evil-owl.el

```diff
@@ -125,10 +127,12 @@ The value may be either 'window or 'posframe."
   "The idle delay, in seconds, before the popup appears."
   :type 'float)
 
-(defcustom evil-owl-register-char-limit nil
-  "Maximum number of characters to consider in a string register."
+(defcustom evil-owl-text-limit nil
```

Comment:
I did a little thinking, and `evil-owl-max-text-length` or `evil-owl-max-string-length` is probably a better name.